src/rendering/custom-element-widget.js

```diff
@@ -19,33 +19,31 @@ function makeDispatchFunction(element, eventName) {
 
 function subscribeDispatchers(element) {
   let {customEvents} = element.cycleCustomElementMetadata;
-  let disposables = new Rx.CompositeDisposable();
+  let dispatcherObservables = [];
   for (let streamName in customEvents) { if (customEvents.hasOwnProperty(streamName)) {
     if (/\$$/.test(streamName) &&
       streamName !== 'vtree$' &&
       typeof customEvents[streamName].subscribe === 'function')
     {
       let eventName = streamName.slice(0, -1);
-      let disposable = customEvents[streamName].subscribe(
+      let obs = customEvents[streamName].doOnNext(
```

Comment:
I don't like to put important side-effects (dispatchCustomEvent) in `doOnNext`. Important ones should be in `subscribe`. Is there any reason why you did this refactor other than code style?